agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1785,24 +1785,17 @@ bool main_thread::handle_1905_autoconfiguration_response(ieee1905_1::CmduMessage
 bool main_thread::send_slaves_enable()
 {
     auto iface_hal = get_wireless_hal();
-    iface_hal->refresh_radio_info();
-    const auto &radio_info = iface_hal->get_radio_info();
 
     for (auto soc : slaves_sockets) {
         auto notification =
             message_com::create_vs_message<beerocks_message::cACTION_BACKHAUL_ENABLE_APS_REQUEST>(
                 cmdu_tx);
 
         if (soc->sta_iface == m_sConfig.wireless_iface) {
-            notification->channel()   = iface_hal->get_channel();
-            notification->bandwidth() = radio_info.bandwidth;
-            notification->center_channel() =
```

Comment:
Since bandwidth and center_channel are not valid, maybe they should be removed from the notification as well for now?

Reply:
these are stubs - and the next step is to utilize them to get full functionality.

at this stage the work done was to port the existing functionality in RDKB - but this change will be one of the first changes to be done to complete the full repeater functionality.

I leave it to you to decide if to remove them from the notification.